src/checkbox.tsx

```diff
@@ -163,8 +163,11 @@ const Checkbox: React.FC<RenderProps | ChildrenProps> = (props) => {
             {props.render ? (
                 props.render(iconCheckbox, labelId)
             ) : (
-                <Inline space={16} alignItems="center">
-                    {iconCheckbox}
+                <Inline space={16}>
+                    {/* Text3 wrapper added to have the same line-height and center checkbox with text */}
+                    <Text3 regular as="div">
+                        <div style={{position: 'relative', top: -2}}>{iconCheckbox}</div>
```

Comment:
can you explain why you need this style?

Reply:
yep, this is to have a perfect pixel centered radio/checkbox with the first line of text (@pladaria and me made some tests and this is the better way to display the item)